apps/meteor/client/views/omnichannel/directory/chats/ChatTable.tsx

```diff
@@ -167,12 +161,26 @@ const ChatTable: FC<{ setChatReload: Dispatch<SetStateAction<any>> }> = ({ setCh
 		[onRowClick],
 	);
 
+	const result = useCurrentChats(query);
+
+	useEffect(() => {
+		setChatReload?.(() => result.refetch);
+	}, [result.refetch, setChatReload]);
+
+	if (result.isLoading) {
+		return <GenericTableLoadingTable headerCells={6} />;
+	}
+	if (result.error) {
+		console.error(result.error);
```

Comment:
do we need to log the errors here?

Reply:
Not needed. Leftovers from local testing. Removed it